tasks/eth/013-fp-upgrade-fjord/README.md

```diff
@@ -0,0 +1,36 @@
+# Mainnet Fjord Fault Proofs Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+This is a playbook for upgrading Fault Proofs on Mainnet for Fjord compatibility.
+
+This sets the `FaultDisputeGame` and `PermissionedDisputeGame` implementations in the `DisputeGameFactory` to newly deployed contracts that have an updated prestate.
+
+Governance post of the upgrade can be found at https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236.
```

Comment:
Let's use a similar format to what we typically have in e.g. https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/010-1-guardian-upgrade

---

<img width="674" alt="image" src="https://github.com/ethereum-optimism/superchain-ops/assets/17163988/eda0feeb-baac-4adb-a4d6-89c1cbb4adae">